src/main/java/frc/robot/commands/CollectorRun.java

```diff
@@ -0,0 +1,48 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Collector;
+
+public class CollectorRun extends CommandBase {
+  private Collector m_subsystem;
+
+  /**
+   * Creates a new CollectorPassive.
+   */
+  public CollectorRun(Collector subsystem) {
+    m_subsystem = subsystem;
+
+    addRequirements(m_subsystem);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    m_subsystem.drive(.5);
```

Comment:
This is where we were working. The next step is to figure out how to read the value of the right stick on the xbox controller in here.